com.unity.shadergraph/Editor/Importers/ShaderSubGraphImporter.cs

```diff
@@ -43,6 +44,7 @@ public override void OnImportAsset(AssetImportContext ctx)
             var subGraphPath = ctx.assetPath;
             var subGraphGuid = AssetDatabase.AssetPathToGUID(subGraphPath);
             graphAsset.assetGuid = subGraphGuid;
+            graphAsset.subGraphObjectIdOverride = "SubGraphImporter_subGraphData_luid_X";
```

Comment:
There is only ever one subGraphData object per GraphAsset in this case. I couldn't find a situation where this caused any problems. To avoid complexity and implied importance of guids, I left this as a magic string.